components/support/migration/metrics.yaml

```diff
@@ -73,6 +73,34 @@ migration.telemetry_identifiers:
       - fenix-core@mozilla.com
     expires: 2020-10-01
     lifetime: ping
+  success_reason:
+    type: counter
+    description: >
+      Why this migration succeeded. See codes in TelemetryHelpers.kt
+    send_in_pings:
+      - migration
+    bugs:
+      - https://github.com/mozilla-mobile/android-components/issues/5653
+    data_reviews:
+      - TODO
```

Comment:
Link to the PR here 👍

Reply:
This needs to be a link to a data review approval comment, hence the TODO :)